bot/exts/christmas/advent_of_code/_cog.py

```diff
@@ -349,3 +353,11 @@ def _build_about_embed(self) -> discord.Embed:
 
         about_embed.set_footer(text="Last Updated")
         return about_embed
+
+    async def cog_command_error(self, ctx: commands.Context, error: Exception) -> None:
+        """Custom error handler if an advent of code command was posted in the wrong channel."""
+        if isinstance(error, InChannelCheckFailure):
+            await ctx.send(f":x: Please use <#{Channels.advent_of_code_commands}> for aoc commands instead, please.")
+            error.handled = True
+        else:
+            raise error
```

Comment:
I don't believe it's desirable to re-raise the  error because I'm concerned it then won't get handled properly by the global command error handler. That aforementioned handler will receive the error without it needing to be raised here.

Reply:
Noted! I was unsure if having the custom cog error handler would stop the error from propagating through. New commit has removed the re-raising of the error.